Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -3,11 +3,14 @@
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
+    const double MSEC_TO_HR = 1 / 1000 / 3600;
+    const double HR_TO_MSEC = 1000 * 3600;
 }
 
 BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
 : initialStateOfChargePercent_(initialStateOfChargePercent)
-{
+, remainingAmphours_(BATTERY_AMP_HOUR_CAPACITY * initialStateOfChargePercent / 100)
```

Comment:
This can be done with the member initialization list as well.

Reply:
Can you please clarify this? I believe that this is already in a member initializer list

Reply:
initialStateofChargePercent is being initialized in the member intializer list.

it could be
```c++
 
  BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
  : initialStateOfChargePercent_(initialStateOfChargePercent)
  , remainingAmphours_(BATTERY_AMP_HOUR_CAPACITY * initialStateOfChargePercent / 100)
...
```

Reply:
Ah ya, you're right, I dunno why I read the diff that it didn't haha mbmb